### DISCUSSION TITLE
宇宙農場上菜啦：太空辣椒成功採收，「史上最佳太空墨西哥捲餅」正式出爐

### DISCUSSION THREAD

**Discussion by 110409538** [1101-GS3073-C/Python-Community-GS3073-C#115] [General]
平凡中的不平凡，太空辣椒成功收成
在地球，種植採收辣椒並不是了不起的事，年復一年都會發生，但10月29日這天，平凡無奇的事卻在國際太空站發生了。

眾目睽睽下，美國太空總署（NASA）太空人范德黑（Mark Vande Hei）小心翼翼伸手，從先進植物培養系統（Advanced Plant Habitat）摘下第一批在太空成長結實的辣椒，並和其他人一起現場品嘗。

現採辣椒不能錯過，太空墨西哥捲餅登場！
但光啃辣椒可無法滿足NASA太空人梅根亞瑟（Megan McArthur），她接著拿出墨西哥烤肉（fajita beef）、復水番茄、菜薊等食材，搭配這次採收的太空辣椒，做成她口中「有史以來最棒的太空墨西哥捲餅」。

部分現場吃，部分送回地球做研究
飽餐一頓後，接下來太空人預計會把一部分太空辣椒送回地球分析，幫助地球科學家進一步了解如何在太空種植作物。
![image](https://user-images.githubusercontent.com/90551832/141675790-03fcf2ae-3889-4e7b-ad62-8c200e761e09.png)
![image](https://user-images.githubusercontent.com/90551832/141675814-37e36240-1d57-4881-b092-62f55aaa94a3.png)
▲ 趁著新鮮的太空辣椒剛採收，梅根亞瑟快手快腳拿出餅皮、墨西哥烤肉，搭配辣椒做了太空墨西哥捲餅。（Source：NASA）

這也太酷了吧,尤其是第二張有墨西哥捲餅飄在空中的照片.

新聞來源:https://technews.tw/2021/11/13/astronauts-have-a-taco-taste-test-using-first-chile-peppers-grown-in-space/

**Comment by chia-kai-chang** [comment]
好酷~

![image](https://user-images.githubusercontent.com/1768891/141676171-db12ced1-0e3f-4c18-9449-4067a4d0d019.png)

**Comment by wh20502** [comment]
生存很重要的食物來源，將來如果要移居外太空，那麼太空糧食來源必定是要解決得，總不可能每個月回地球補貨哈哈。也許哪天我們會重返地球，種披薩植物。

**Reply by minniewei** [reply]
我覺得這是很有遠見的想法，也期待這項技術在未來能更加精熟

**Comment by Shiba010** [comment]
這樣能算是超級無底宇宙食材嗎？XD

**Comment by cobb110202008** [comment]
這項技術的重大突破，使人類有機會在外太空能種植糧食作物。

**Comment by yasklop** [comment]
看來太空餐廳不遠了！

**Comment by jack90325** [comment]
那個辣椒也太大顆，看起來是給了很多肥料，家裡種的都沒那麼大
